src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,15 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Email;
+import javax.validation.constraints.Size;
+
 public class UserRequestDto {
+    @Email
     private String email;
+    @Size(min = 5)
     private String password;
+    @Size(min = 5)
+    private String repeatPassword;
```

Comment:
think of more checks and let's not add repeatPassword if you are not checking if it's the same as password